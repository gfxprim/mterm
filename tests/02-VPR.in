3 3
\e[8e
