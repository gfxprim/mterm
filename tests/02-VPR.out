 ---
|   |
|   |
|   |
 ---
size 3x3 cursor 0x2
